eth/interop.go

```diff
@@ -27,7 +27,16 @@ func (s *Ethereum) CurrentInteropBlockTime() (uint64, error) {
 	if chainConfig.InteropTime == nil {
 		return 0, errors.New("interop time not set in chain config")
 	}
-	return s.BlockChain().CurrentBlock().Time, nil
+	// The pending block may be aliased to the current block in op-geth. Infer the pending time instead.
+	currentHeader := s.BlockChain().CurrentHeader()
+	penultimateHeader := s.BlockChain().GetHeaderByNumber(currentHeader.Number.Uint64() - 1)
```

Comment:
Is it possible for this to underflow when the current header is 0? I think its unlikely in practice, but possible (especially in tests)

Reply:
Good catch. At genesis, it won't be possible to infer the block time. Should we fall back to using a block time of `0` if our head is at genesis?

> we should have some error handling for this math, abstracted as some helper function (which can live here in this file) called inferBlockTime() error

I've gone ahead with this for now, returning an error in the genesis case.